### ISSUE TITLE
Only Describe images with empty Alt=""

### ISSUE THREAD

**Issue by clapierre** [ways2read/fido-ai#22]
Feature Request
I have a word document with 90% of the images already described, I would like Fido to only attempt to describe those with missing alt text descriptions.

Maybe even an override to describe those also marked as presentational.

**Comment by ways2read** [comment]
Thanks for the feature request. I also encounter images with alt text such as 'image' or 'image99.jpg'. Perhaps there could be an option to skip generating image description if alt text is missing or length is < 15 characters?

**Comment by ways2read** [comment]
This feature is implemented in code but not yet in the UI.

**Comment by ways2read** [comment]
Implemented in the Word add-in

**Comment by clapierre** [comment]
Were you able to detect if the image description was a file name?  Something ending in .xyz? or a single word?

**Comment by ways2read** [comment]
It skips any image marked Decorative. It treats an image as already described if AlternativeText (trimmed) is longer than 15 characters.

Not very sophisticated, but typically I see fake alt text as image76.jpg or similar.
